value/src/test/java/com/google/auto/value/processor/PropertyAnnotationsTest.java

```diff
@@ -99,7 +99,8 @@ private JavaFileObject sourceCode(List<String> imports, List<String> annotations
     return JavaFileObjects.forSourceLines("foo.bar.Baz", lines);
   }
 
-  private JavaFileObject expectedCode(List<String> annotations) {
+  private JavaFileObject expectedCode(List<String> annotations, String expectedConstructorParamAnnotation) {
+    String constructorParamPrefix = expectedConstructorParamAnnotation != null ? expectedConstructorParamAnnotation + " " : "";
```

Comment:
Google style imposes a 100-column limit. Can you split this line? There might be a long line in CompilationTest too.